build-llvm34.md

```diff
@@ -23,6 +23,12 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
    $ export CPLUS_INCLUDE_PATH=/usr/include/x86_64-linux-gnu
    ```
 
+   **(Optional) Build KLEE with TCMalloc support:** By default, KLEE uses malloc_info() to get memory usage, which limits the maximum memory to 2000MB. To avoid this limitation, it is possible to also configure KLEE with TCMalloc. It it thus necessary to install TCMalloc:
```

Comment:
... KLEE uses `malloc_info()` to observe and to restrict its memory usage. Due to limitations of `malloc_info()` [maybe a link here], the maximum limit set is 2 GB. To support bigger limits, KLEE can use TCMalloc as an alternative allocator.
It *is* ..."